### DISCUSSION TITLE
Add addtioanl calendar?

### DISCUSSION THREAD

**Discussion by KHALED-LAKEHAL** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2754] [Q&A]
Hello how can use different calendar style like hijri and Persian calendar in calendar without change ui culture info.

**Comment by Keboo** [comment]
For setting it on a single control, you can set the `Language` property on the control to change the UI language for just that element.

**Reply by KHALED-LAKEHAL** [reply]
Thanks for replying i set this property to calendar but this property works only on header text and CurrentDateTextBlock text but in  PART_MonthView or  PART_YearView

**Reply by Keboo** [reply]
This looks like it is likely a bug.

**Reply by KHALED-LAKEHAL** [reply]
Thanks this bug but related with wpf library because calendar supported only gregorian calendar and when set language to ar-SA this property not worked correctly.
